template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazBaseConfigProvider.java

```diff
@@ -31,6 +37,15 @@ public String getServiceType() {
 
     @Override
     protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, TemplatePreparationObject source) {
+        if (source.getProductDetailsView() != null && isRazTokenConfigurationSupported(source.getProductDetailsView().getCm())) {
+            if (CloudPlatform.AZURE == source.getCloudPlatform()) {
+                String safetyValveValue = getSafetyValveProperty("ranger.raz.bootstrap.servicetypes", "adls");
+                return List.of(config(RANGER_RAZ_SITE_XML_ROLE_SAFETY_VALVE, safetyValveValue));
+            } else if (CloudPlatform.AWS == source.getCloudPlatform()) {
+                String safetyValveValue = getSafetyValveProperty("ranger.raz.bootstrap.servicetypes", "s3");
+                return List.of(config(RANGER_RAZ_SITE_XML_ROLE_SAFETY_VALVE, safetyValveValue));
+            }
```

Comment:
The logics in these two conditions are almost the same, apart from the property value. Perhaps this part could be simplified to reduce redundancy.